code/datums/status_effects/buffs.dm

```diff
@@ -74,8 +74,8 @@
 
 /datum/status_effect/blooddrunk
 	id = "blooddrunk"
-	duration = 10
-	tick_interval = -1
+	duration = 1 SECONDS
+//	tick_interval = -1 
```

Comment:
Unless the code comments for status effects are wrong, this fundamentally does nothing, except make this status effect tick every second.
https://github.com/Bubberstation/Bubberstation/blob/e7b7473c71e4bac488bdfea6f00339165423c7db/code/datums/status_effects/_status_effect.dm#L6-L14

As the duration is in deciseconds, 10 already equals `1 SECOND` (In fact, the SECOND(S) macro just multiplies whatever number by 10.)
The blooddrunk status effect also doesn't need to tick, as it's a "get a speed boost when you get it, lose it when you lose it" type thing.

In short, this does nothing.
Also, please test changes.

Reply:
I have tested and for me it works fine, less the somewhat stun immunity part and damage tankiing sometimes works, sometimes it doesn't. I have the feeling it has to do with server lag. Maybe increasing the ticks should resolve it to give a false time duration I try to make it truly work.

Reply:
I'll try increasing the tick duration.

Reply:
> I have tested

I mentioned that because 
![image](https://github.com/user-attachments/assets/a41ca407-c593-44cc-bbae-97140bfa6907)

Also, you can PR this to TG itself - as is, one of the maints is almost definitely going to, at minimum, ask you to mark the change, if not make it modular (or port it to TG anyways)

Reply:
I'm not gonna wait for an upstream PR to have it fixed, how can I modularize an already existing status effect just to manipulate its time duration though?